src/components/navigation/NavigationBar.tsx

```diff
@@ -29,7 +31,10 @@ export default function NavigationBar(): ReactElement {
   const location: { pathname: string } = useLocation();
 
   const [current, setCurrent] = useState<Pages>(Pages.Home);
+  const firebase = useFirebase();
+  const authUser = useAuthUser();
 
+  // TODO: isActive is not highlighting the active page when the browser back button is pressed
   const isActive = (key: Pages): CSSProperties => ({
     fontWeight: current === key ? 'bold' : 'normal',
```

Comment:
Nice catch, could we create a bug ticket and link it?

Reply:
Yup, I've created https://promopal.atlassian.net/browse/PP-101 for it